hack/verify-generated-docs.sh

```diff
@@ -22,8 +22,17 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
 kube::golang::setup_env
+binaries=(
```

Comment:
extreme nit: add a space before this line to make the diff match the diff of hack/update-generated-docs.sh